pkg/tfbridge/property_path.go

```diff
@@ -125,3 +155,23 @@ func willTriggerReplacementRecursive(
 
 	return replacement
 }
+
+func schemaContainsComputed(
+	path propertyPath, rootTFSchema shim.SchemaMap, rootPulumiSchema map[string]*info.Schema,
+) bool {
```

Comment:
As implemented, I would call this `pathContainsComputed`, and make it clear that the function checks if any path prefixed by `path` is marked as computed.